backend/inspirehep/search/mappings/v7/records/journals.json

```diff
@@ -245,10 +245,25 @@
       "normalizer": {
         "lowercase_normalizer": {
```

Comment:
this affects other fields too that are using it. Maybe create a new specific normalizer for this instead?

Reply:
I was thinking about it, it's only affecting `_collections` which will not be affected but I agree it's cleaner to separate it.